labeler/src/main.rs

```diff
@@ -51,4 +58,28 @@ fn main() {
             }
         }
     }
+
+    let issues_in_triage =
+        crate::github::get_labeled_issues(&config, "rust-lang/glacier", "triage".to_string())
+            .unwrap();
+    for i in closed_issue_numbers {
+        let title = format!("Issue {}", i);
+        if issues_in_triage
+            .iter()
+            .any(|issue| issue.title.starts_with(&title))
+        {
+            // Triage issue already created
+            continue;
+        }
+
+        let body = format!("See rust-lang/rust#{id}", id = i);
+
+        crate::github::create_issue(
+            &config,
+            &format!("Issue {} has been closed", i),
```

Comment:
```suggestion
            &format!("issue-{} has been closed", i),
```

I'd prefer this format.